guides.md

```diff
@@ -28,6 +29,145 @@ our values.
 - [Voicemail](#voicemail)
 - [References](#references)
 
+## Email
+
+### Creating new inboxes (Administrators)
+
+What you'll need:
+- Admin access to [our Mailcow email server](https://link.hypha.coop/email)
+- The following user info (from https://link.hypha.coop/employees):
+    - preferred email handle@hypha.coop
+    - preferred value for "Full Name"
+    - primary email (where they'd like @hypha.com emails to be forwarded initially)
+    - Signal-friendly phone number
+
+_(Consider creating a [new GitHub issue in `hyphacoop/organizing`](https://github.com/hyphacoop/organizing/issues/new) to track email onboarding.)_
+
+1. Sign into Mailcow: https://link.hypha.coop/email
+2. Navigate to "Mailboxes" tab
+3. Click "+Add mailbox" button at bottom of screen
+    - If the namespace is already taken by an alias, ensure it's not in use before deleting it.
+4. Fill out the following fields, noting the password:
+    - Username: info provided by new user
+    - Domain: `hypha.coop`
+    - Full name: info provided by new user
+    - Quota: `4096`
+    - Password: auto-generated
+    - Active: checked :ballot_box_with_check:
```

Comment:
The way I have been doing "value for key in the guide is to put upticks around all the values. Suggesting to share this practice throughout for consistency and ease to recognize from line to line.

```suggestion
4. Fill out the following fields, noting the password:
    - Username: info provided by new user
    - Domain: `hypha.coop`
    - Full name: `info provided by new user`
    - Quota: `4096`
    - Password: `auto-generated`
    - Active: `checked` :ballot_box_with_check:
```

Reply:
I'm down for this!

Though holding some space for (and being ok with) messiness and inconsistency has entropic value and benefit as well, and I might even suggest "messiness" be a future value we consider :) https://github.com/hyphacoop/organizing/issues/234#issuecomment-600188610

Reply:
Also, without "checked" written out, "unchecked" doesn't work as well, and not sure if emoji are proper for a11y on those

Reply:
If going this way, I guess this

```suggestion
4. Fill out the following fields, noting the password:
    - Username: `[info provided by new user]`
    - Domain: `hypha.coop`
    - Full name: `[info provided by new user]`
    - Quota: `4096`
    - Password: `[auto-generated]`
    - Active: checked :ballot_box_with_check:
```

Reply:
>"messiness" be a future value we consider

you know what my vote will be :)

Reply:
>Password: `[auto-generated]`

If `auto-generated` is the field value then it shouldn't have `[]`

Reply:
It's a little helper link, if that makes sense.

<img width="1049" alt="Screen Shot 2020-03-17 at 3 23 12 PM" src="https://user-images.githubusercontent.com/305339/76893739-3ea0d480-6863-11ea-99ea-645ac21af9a3.png">

Reply:
Added this screenshot to this section of guide, with others.